episodes/2-keras.Rmd

```diff
@@ -410,6 +410,18 @@ Keras distinguishes between two types of weights, namely:
 If these reasons are not clear right away, don't worry! In later episodes of this course, we will touch upon a couple of these concepts.
 ::: 
 
+
+::: instructor
+Optional challenge number 3 (see below; for visualizing the model) requires the command-line tool `dot` from Graphviz.
+If `dot` is missing, you can either:
+
+- show the solution, or
+- try to install it.
+
+The [installation instructions for Graphviz can be found here](https://graphviz.org/download/).
```

Comment:
```suggestion
For optional question 3 in below's challenge 'Visualizing the model' the challenge is to visualize the network. You could choose to show and discuss the resulting visualization to the learners, so also learners that did not finish the optional exercise can learn from the visualization of the model.
```

See my suggestion below. Do you agree we can assume that the installation was successful?

Reply:
Fixed in 5f03bf3 and I confirm the installation in a fresh environment. cc: @carschno

Reply:
Great, thank you so much!
(I am just taking over for the moment because Sven does only is back in the office next week).